src/components/commandHandler.ts

```diff
@@ -138,7 +130,8 @@ export abstract class CommandHandler {
     }
 
     static IdentifyHandler(): void {
-        if (!this.playbackManager.isPlaying()) {
+        if (!PlaybackManager.isPlaying()) {
+            DocumentManager.setAppStatus('waiting');
```

Comment:
When I tried latest master on jellyfin/jellyfin and jellyfin/jellyfin-web, The identify command was sent from the client while buffering  (after load). isPlaying() returned false here and the receiver went back to "waiting" state  (i.e. "welcome screen") while the video played behind it. Got the same behavior every time I tried. When I removed these actions, it works as expected.

Should this logic really be in the identify command handler? For me, this is looks like side effect. I think more robust guards than isPlaying check must be in place if this is kept.

Reply:
What do you think of [this](https://github.com/hawken93/jellyfin-chromecast/pull/228/commits/323983aca5a3f2173cd05406d5398831c8e0f153) solution?

Reply:
Looks promising and I am happy with it if it works.